nixos/modules/config/xdg/terminal-exec.nix

```diff
@@ -0,0 +1,64 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.xdg.terminal-exec;
+  inherit (lib) mkIf mkOption types;
+in
+{
+  meta.maintainers = with lib.maintainers; [ Cryolitia ];
+
+  ###### interface
+
+  options = {
+
+    xdg.terminal-exec = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Whether to enable the proposal for XDG terminal execution utility.
+        '';
+      };
```

Comment:
enable options are normallly done with `lib.mkEnableOption`
but idk about this one